cli/src/cli/run.rs

```diff
@@ -100,13 +100,17 @@ pub fn run(
             n3h_bootstrap.push(n3h_bootstrap_node.unwrap())
         }
 
+        // Load end_user config file
+        let n3h_end_user_config_filepath = env::var("HC_N3H_SETTINGS_FILE").ok();
```

Comment:
We started down this path of naming things n3h, but I'd actually like to undo that. The things in this config are going to need to exist even when we write the embedded rust version of the networking module. So, maybe `"NETWORKING_CONFIG_FILE"`?

Reply:
changing just this one for now.